src/Microsoft.ML.Data/Utils/ApiUtils.cs

```diff
@@ -16,14 +17,15 @@ namespace Microsoft.ML
 
     internal static class ApiUtils
     {
-        private static OpCode GetAssignmentOpCode(Type t)
+        private static OpCode GetAssignmentOpCode(Type t, params Attribute[] attributes)
```

Comment:
This doesn't need to be an array anymore - it can be an IEnumerable - so the upstream callers don't need to call `ToArray`